app/shared-components/PromoModal/PromoModal.jsx

```diff
@@ -76,7 +76,7 @@ class PromoModal extends React.Component {
 				break;
 			case INSIGHTS:
 				sendMessage('ping', 'promo_modals_insights_upgrade_cta');
-				url = `https://checkout.${DOMAIN}.com/insights?utm_source=gbe&utm_campaign=${utm_campaign}`;
+				url = `https://www.ghostery.${DOMAIN}.com/insights/?utm_source=gbe&utm_campaign=in_app_upgrade=${utm_campaign}`;
```

Comment:
this will expand to `https:///www.ghostery.ghostery.com/insights/?utm_source=gbe&utm_campaign=in_app_upgrade=<utms>` take a look at the value of DOMAIN from line 26 of this file. as this URL does not change depending on environment we should probably not be using any conditional domains for this URL